src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let price = 0;
+
+  if (days < 3) {
+    price = days * 40;
```

Comment:
move numbers to variables with meaningful names. It'll make the code clearer. Now it's not clear what is 40, 20, 50.